etc/kayobe/inventory/group_vars/ceph/firewall

```diff
@@ -0,0 +1,83 @@
+---
+
+firewallgen_enable_firewall: True
+
+firewallgen_ipv4_input_allow_rules:
+  - interface: "all"
```

Comment:
I don't think this is correct, although certainly that is what the service is doing.

I think it should be storage management net only, but... you know, I am probably wrong.